.moai/release/RELEASE-NOTES-v2.15.0.md

```
@@ -0,0 +1,289 @@
+| SPEC ID | Title | Status | Impact |
+|---------|-------|--------|--------|
+| SPEC-V3R3-DEF-007 | Convention Compliance Sweep | Implemented | 11 skills PD, manager-git scope boundaries |
+| SPEC-V3R3-DEF-001 | ORC Dependency Cycle Resolution | Implemented | D-CRIT-001 resolved, WF-001/MIG-001 one-way |
+| SPEC-V3R3-ARCH-003 | Expert Agent Tool Uplift | Implemented | 7 expert agents + escalation protocol |
+| SPEC-V3R3-ARCH-007 | Token Circuit Breaker | Implemented | Per-agent budgets, 75%/90% thresholds |
+| SPEC-V3R3-COV-001 | Mobile Native Coverage | Implemented | expert-mobile + 3 mobile skills |
+| SPEC-V3R3-CMD-CLEANUP-001 | Commands Cleanup | Implemented | /moai gate added, context removed |
+
+### 새 기능
+
+#### Convention Compliance Sweep (SPEC-V3R3-DEF-007)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)